src/components/navbar.module.css

```
@@ -70,14 +80,90 @@
+    .title, 
+    .link, 
+    .contributor {
+        font-size: 13px;
+        flex-direction: row;
+        justify-content: center;
+
+    }
+    .link {
+        margin-left: 2rem; 
+        margin-top: 0;
+        margin-left: 300px; 
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

margin-left is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)